src/current/v23.1/cdc-queries.md

```diff
@@ -190,6 +190,57 @@ For newly inserted rows in a table, the `cdc_prev` column will emit as `NULL`.
 If you do not need to select specific columns in a table or filter rows from a changefeed, you can instead create a changefeed using the [`diff` option]({% link {{ page.version.version }}/create-changefeed.md %}#diff-opt) to emit a `before` field with each message. This field includes the value of the row before the update was applied.
 {{site.data.alerts.end}}
 
+### Reference TTL in a CDC query
+
+In CockroachDB, table row deletes occur as a result of regular SQL transactions or through [row-level TTL]({% link {{ page.version.version }}/row-level-ttl.md %}). When your changefeed emits [delete event messages]({% link {{ page.version.version }}/changefeed-messages.md %}#delete-messages), you may need to distinguish between these two types of deletion. For example, only emitting messages for row-level TTL deletes from your changefeed.
```

Comment:
suggest making 'regular SQL transactions' a link to the delete data docs: https://www.cockroachlabs.com/docs/stable/delete-data - esp since that page has a 'performance considerations' section https://www.cockroachlabs.com/docs/stable/delete-data#performance-considerations